src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1399,6 +1400,29 @@ void VM_Version::get_processor_features() {
     if (FLAG_IS_DEFAULT(AllocatePrefetchInstr) && supports_3dnow_prefetch()) {
       FLAG_SET_DEFAULT(AllocatePrefetchInstr, 3);
     }
+#ifdef COMPILER2
+    if (FLAG_IS_DEFAULT(ArrayCopyPartialInlineSize) ||
+        (!FLAG_IS_DEFAULT(ArrayCopyPartialInlineSize) &&
+         ArrayCopyPartialInlineSize != 0 &&
+         ArrayCopyPartialInlineSize != 32 &&
+         ArrayCopyPartialInlineSize != 64)) {
+      int pi_size = 0;
+      if (MaxVectorSize > 32 && AVX3Threshold == 0) {
+        pi_size = 64;
+      } else if (MaxVectorSize >= 32) {
+        pi_size = 32;
+      }
+      if(!FLAG_IS_DEFAULT(ArrayCopyPartialInlineSize)) {
+        warning("Setting ArrayCopyPartialInlineSize as %d", pi_size);
+      }
+      ArrayCopyPartialInlineSize = pi_size;
+    }
+
+    if (ArrayCopyPartialInlineSize > MaxVectorSize) {
+      ArrayCopyPartialInlineSize = MaxVectorSize;
+      warning("Setting ArrayCopyPartialInlineSize as MaxVectorSize");
```

Comment:
warning only if ArrayCopyPartialInlineSize is not default.

Reply:
I don't see your fix for my comment. I asked to add `if(!FLAG_IS_DEFAULT(ArrayCopyPartialInlineSize))` check

Reply:
Default value for ArrayCopyPartialInlineSize = -1 with a value range [-1,64], default value for MaxVectorSize=0 with a value range [0,max_int]; control flow will be reaching to this warning only for a non-default value of ArrayCopyPartialInlineSize.